client/my-sites/earn/ads/form-settings.tsx

```diff
@@ -176,6 +206,8 @@ const AdsFormSettings = () => {
 						onChange={ handleChange }
 						disabled={ isLoading }
 						label={ translate( 'Run ads only for logged-out users (less revenue)' ) }
+						// className undefined to resolve TS warning
+						className={ undefined }
```

Comment:
huh? 🤔 `FormRadio` requires className, why? Could it be made optional at component level instead?

Reply:
@simison - Yes, we could (and probably should) refactor some of these underlying components to not require className, and I considered doing so here.

BUT... it would dramatically expand the reach and risk of this PR if it included refactoring underlying, common Automattic components, which are themselves used throughout the code base. My goal in each of these refactors is to keep the size/risk small. If we felt it was worth updating the underlying components, I'd rather do that in separate PRs (one per component that needs refactoring). 

As a side note, one reason this appears now is that a lot of those common components are in JS, and many of the components that use them throughout the codebase are also in JS. This is just one of those things that gets revealed when upgrading to TS.

Reply:
Just to follow convention with other uses of this component, I've updated the PR to just comment out the TS errors for now.